### ISSUE TITLE
Migrate the guide markdown.template to Bootstrap / UI components

### ISSUE THREAD

**Issue by sanand0** [gramener/gramex#57]
- Source: <https://code.gramener.com/cto/gramex/issues/622>
- Author: [s.anand](https://code.gramener.com/s.anand)
- Created: Mon 04 Feb 2019 06:43

**Comment by richielionell** [comment]
Code snippets seem to be hidden because of a white background. The code is visible only when I click the copy code icon.

![guide-code-snippet](https://user-images.githubusercontent.com/4058058/52420306-d6d47200-2b17-11e9-8015-0dd73ed26527.gif)

**Comment by sanand0** [comment]
The code is now visible.

**What happened?**

On gramener.com, I found that `monokai.css` (the file that has the code styling) had zero bytes.

```
ubuntu@www:/mnt/gramener/apps/v1/guide$ ls
...
-rw-rw-r--  1 ubuntu ubuntu     0 Feb  8 06:26 monokai.css
...
```

The [build](https://code.gramener.com/cto/gramex/-/jobs/98743) was successful. The file had the correct size in the source directory `/mnt/gramener/apps/gramex/gramex/apps/guide` as well.

I [re-ran the build](https://code.gramener.com/cto/gramex/-/jobs/98819). Issue persisted.

I manually copied `cp /mnt/gramener/apps/gramex/gramex/apps/guide/monokai.css /mnt/gramener/apps/v1/guide/` and things are fine now.

So, no idea what the problem is. It might recur. Let's be on the lookout.

**Comment by sanand0** [comment]
This problem recurs on deploy.  Solution:

```bash
# Run this on gramener.com
cp /mnt/gramener/apps/gramex/gramex/apps/guide/monokai.css /mnt/gramener/apps/v1/guide/
```
On the browser, visit https://learn.gramener.com/guide/monokai.css?v=2 (or whatever the browser requests) and refresh via Ctrl-Shift-R.

**Comment by pratapvardhan** [comment]
FYI -- This reoccurred. I moved `monokai.css`.

**Comment by sanand0** [comment]
Fixed via 9ec129c5

`monokai.css` is dynamically generated by `setup.sh`. We don't need that.
